{
  "name": "python-zope-exceptions",
  "version": "5.0.1-3",
  "depends": [
    "python-setuptools",
    "python-zope-interface"
  ],
  "makedepends": [
    "python-setuptools",
    "python-zope-interface"
  ],
  "receive": 196709240,
  "transmit": 468735,
  "disk": 280737
}
